src/request.rs

```diff
@@ -1,15 +1,15 @@
 use std::collections::BTreeMap;
 
 use ruma::identifiers::UserId;
-use ruma_client::{Error, HttpsClient};
+use ruma_client::{Client, Error};
 
 /// A builder for a request to the Matrix homeserver.
 #[derive(Debug, Clone)]
 pub struct RequestBuilder<'a, R>
 where
     R: ruma::api::OutgoingRequest + std::fmt::Debug,
 {
-    client: &'a HttpsClient,
```

Comment:
I'm a bit confused if we have to now have to enable `tls` at build time. (https://github.com/ruma/ruma/commit/50eb700571480d1440e15a387d10f98be8abab59)
But I guess I will see this soon enough :P